pkg/executor/simple.go

```diff
@@ -96,6 +96,13 @@ type SimpleExec struct {
 	staleTxnStartTS uint64
 }
 
+type resourceOptionsInfo struct {
+	maxQueriesPerHour     int64
```

Comment:
Are the `maxQueriesPerHour` and the other 2 resource limits required?

Reply:
These three parameters are reserved and are not currently implemented

Reply:
IMHO if we do not have an explicit plan supporting them, they can be removed?

Reply:
These three resource options has existed but not valid.  I have added comments for them.